packages/sdk/src/credential/teamsUserCredential.browser.ts

```diff
@@ -132,6 +132,8 @@ export class TeamsUserCredential implements TokenCredential {
   /**
    * Get access token from credential.
    *
+   * Important: Graph access tokens are stored in sessionStorage, read more here: https://aka.ms/teamsfx-session-storage-notice
```

Comment:
Maybe no need to mention **Graph** access tokens? as all access tokens besides sso token are stored at session storage.

Reply:
Sure, updated